image/Archive_Tar/Dockerfile.archive_tar

```diff
@@ -0,0 +1,46 @@
+################################################################################
+# Container image for ease of testing Archive_Tar vulnerabilty
+# and how it is mitigated under WebAssembly
+# More info:
+#   * https://www.drupal.org/sa-core-2020-013
+#   * https://nvd.nist.gov/vuln/detail/CVE-2020-28949 
+################################################################################
+ARG IMAGE_REPOSITORY=docker.io
+
+FROM $IMAGE_REPOSITORY/library/ubuntu:22.04
+RUN apt-get update && apt-get install wget curl unzip python3 software-properties-common -y
+
+# PHP 8.2
+RUN add-apt-repository ppa:ondrej/php
+RUN apt update
+RUN apt install php8.2-cli php-pear -y
```

Comment:
@gzurl I would about to suggest installing the interpreters using existing images like `bitnami/python` and `bitnami/php`:

```Dockerfile
FROM bitnami/python:3.9.16 AS python
FROM bitnami/php-fpm:8.2.6 AS php

FROM bitnami/minideb:latest

COPY --from=python /opt/bitnami/python /opt/python
COPY --from=php /opt/bitnami/php /opt/php

ENV PATH="${PATH}:/opt/python/bin:/opt/php/bin"
```

However, it doesn't work as the interpreters rely on some shared libraries. In that case, it's better to use your approach as the installation will take care about installing them.

I wanted to leave the comment just it case it helps you in the future :)